README.md

```diff
@@ -161,38 +177,46 @@ console.log(newData);
 
 ## Development
 
-To run tests
+To run tests on your project in development, use the following command:
 
 ```
 npm run test
 ```
 
 ### vc-test-suite
 
-You can run the vc-test-suite against `open-attestation` by running `npm run test:vc`. This command will:
+You can run `vc-test-suite` against `open-attestation` by using the `npm run test:vc` command. It will:
 
 - clone https://github.com/w3c/vc-test-suite.git
 - copy the local configuration (`vc-test-suite-config.json`) into the cloned repository
 - install the latest version of `@govtechsg/open-attestation-cli`
-- monkey patch `open-attestation` in `@govtechsg/open-attestation-cli`. That means that the current version of the project will be built and replace the one installed with `@govtechsg/open-attestation-cli`.
+- monkey patch `open-attestation` in `@govtechsg/open-attestation-cli`
+  
+  "Monkey patch" means it will build the current version of the project, which will replace the one installed with `@govtechsg/open-attestation-cli`.
+
+#### Local debugging
+
+If you face a problem with one test and want to debug locally:
+
+1. Ensure the `vc-test-suite` folder is available from the root of the project. 
 
-#### Local debug
+    If not, run `npm run test:vc` first and perform the test again.
 
-In the event you face a problem with one test and want to debug locally:
+1. Open `runVcTest.sh` and update `install_vc_test_suite=true` to `install_vc_test_suite=false`. 
 
-1. Ensure the folder `vc-test-suite` is available from the root of the project. If that's not the case, run `npm run test:vc` first.
-1. Open `runVcTest.sh` and update `install_vc_test_suite=true` to `install_vc_test_suite=false`. This line will help to preserve the `vc-test-suite` folder untouched.
+    This line will preserve the `vc-test-suite` folder untouched.
 
-You can now debug from the `vc-test-suite` folder the way you need it.
+You can now debug from the `vc-test-suite` folder the way you need.
 
 ## Additional information
 
-- Found a bug? Have a question? Want to share an idea? Reach us at our [Github repository](https://github.com/Open-Attestation/open-attestation).
-- We are currently building a new version of the schema, compatible with W3C VC. This is very experimental and whatever is available for v2 documents are also available for v3 documents:
+- If you find a bug, have a question, or want to share an idea, reach us at our [Github repository](https://github.com/Open-Attestation/open-attestation).
```

Comment:
```suggestion
- If you find a bug, have a question, or want to share an idea, you can reach us at our [Github repository](https://github.com/Open-Attestation/open-attestation).
```

Reply:
"You can do something" and "Do something" are used interchangeable in the documents. It looks like other product documentation also use "contact us today" or "order it now" as a call for action. Perhaps we don't have to say "you can..." every time for simplicity. Thoughts?